pkg/api/types.go

```diff
@@ -239,6 +241,18 @@ type SecretVolumeSource struct {
 	Target ObjectReference `json:"target"`
 }
 
+// NFS represents and NFS Mount that lasts the lifetime of a pod
+type NFSVolumeSource struct {
+	// Server is the hostname or ip address of the NFS server
+	Server string `json:"server"`
+
+	// SourcePath is the exported NFS share
+	SourcePath string `json:"sourcePath"`
+
+	// MountOptions is a comma delimited list of mount options
+	MountOptions string `json:"mountOptions"`
```

Comment:
Can you put examples of what someone might pass here?  It feels like it should be []string